main.py

```
@@ -258,6 +258,27 @@ async def handler(data):
         elif msg_type == "chain_request":
             start_index = payload.get("start_index", 0)
             limit = payload.get("limit", 500)
+            if (
+                not isinstance(start_index, int)
+                or isinstance(start_index, bool)
+                or start_index < 0
+            ):logger.warning(
```

Ruff (0.16.1)

[error] Multiple statements on one line (colon)

(E701)